lms/static/lms/js/require-config.js

```diff
@@ -64,15 +64,20 @@
             "tags-annotator": 'js/vendor/ova/tags-annotator',
             "diacritic-annotator": 'js/vendor/ova/diacritic-annotator',
             "flagging-annotator": 'js/vendor/ova/flagging-annotator',
-            "jquery-Watch": 'js/vendor/ova/jquery-Watch',
+            "jquery-Watch": "js/vendor/ova/jquery-Watch",
```

Comment:
I think this line got moved into the above section that uses double quotes, and was then moved back here and wasn't updated. The inconsistency in this file drives me crazy!